src/components/DisbursementInviteMessage/index.tsx

```diff
@@ -0,0 +1,156 @@
+import { Fragment, useEffect, useState } from "react";
+import { Card, RadioButton, Textarea } from "@stellar/design-system";
+import { useDispatch } from "react-redux";
+
+import { useRedux } from "hooks/useRedux";
+import { AppDispatch } from "store";
+import { getOrgInfoAction } from "store/ducks/organization";
+
+import "./styles.scss";
+
+interface DisbursementInviteMessageProps {
+  draftMessage?: string;
+  isEditMessage: boolean;
+  onChange?: (message: string) => void;
+}
+
+export const DisbursementInviteMessage = ({
+  draftMessage,
+  isEditMessage,
+  onChange,
+}: DisbursementInviteMessageProps) => {
+  console.log(draftMessage);
+  enum radioValue {
+    ORGANIZATION = "organization",
+    CUSTOM = "custom",
+  }
+
+  const standardOrgMessage =
+    "You have a payment waiting for you from the {{.OrganizationName}}. Click {{.RegistrationLink}} to register.";
+  const customMessagePlaceholder =
```

Comment:
It looks like this placeholder is used as a value instead of a placeholder. If we want to use it as a value, we need to make sure the value will be shown correctly when switching between default and custom messages. Right now, it works only on the first switch.